### ISSUE TITLE
'Continue Listening' disappears from home menu

### ISSUE THREAD

**Issue by dddddddavid** [jellyfin/jellyfin#3045]
'Continue Listening' is enabled in the user home menu but does not appear on the home page for either audiobooks or music.

 - OS: Raspbian

 - Clients: Android / ubuntu / Ubuntu touch
 - Browser: All
 - Jellyfin Version:  10.5.4

 - Installed Plugins:Anime audiodb musicbrains
 - Reverse Proxy: Apache
 - Base URL: no

 - Storage: local

**To Reproduce**

1. Go to jellyfin using any browser
2. Look for 'continue listening'
3. ...
4. File bug report

**Expected behavior**
Continue listening to be available for audiobooks and music somewhere on the home screen

**Logs**

**Screenshots**

**Additional context**

**Comment by Artiume** [comment]
What library type did you use for the audio books?

**Comment by dddddddavid** [comment]
M4b. They play fine. They appear in my audiobooks folder. I just lose my place because contue listening has disappeared for everything

**Comment by Artiume** [comment]
I meant when you created the audio book library in jellyfin. Did you specify music or books

**Comment by dddddddavid** [comment]
Sorry. Books.

**Comment by Artiume** [comment]
https://github.com/jellyfin/jellyfin/issues/2889

You should be seeing them for the audio books, odd that it's not working. My version of nightly is working for them. 

Music doesn't preserve the continue listening. My pr references the enhancement to toggle it for any library you want.

**Comment by Artiume** [comment]
You should obfuscate your url's

**Comment by dddddddavid** [comment]
Noted

**Comment by dddddddavid** [comment]
OK. Continue watching (for the books) reappeared after an all day absence. I was able to play them on desktop & and on mobile (Ubuntu Touch - Morph browser/ webapp). After closing on the phone, the most recently listened-to audiobook vanished again from 'continue listening' on desktop and mobile browser. The other three I tested remained.

Able to start the book (from the beginning) from Latest Audiobooks, but no blue progress bar on the thumbnail (using desktop browser - firefox 75). Pressed stop, but the book doesn't appear in recently watched.

Book starts from the beginning again when launched from 'latest audiobooks'
